script/results.html

```diff
@@ -18,22 +31,45 @@
 
     <ul class="nav nav-tabs">
       <li class="nav-item">
-        <button class="nav-link active" id="config-tab" data-bs-toggle="tab" data-bs-target="#config" type="button" role="tab" aria-controls="config" aria-selected="true">Configuration</button>
+        <button class="nav-link active" id="config-tab" data-bs-toggle="tab" data-bs-target="#config" type="button" role="tab" aria-controls="config" aria-selected="true" onclick="disableButton2()">Configuration</button>
       </li>
       <li class="nav-item">
-        <button class="nav-link" id="metadata-tab" data-bs-toggle="tab" data-bs-target="#metadata" type="button" role="tab" aria-controls="metadata" aria-selected="false">Metadata</button>
+        <button class="nav-link" id="metadata-tab" data-bs-toggle="tab" data-bs-target="#metadata" type="button" role="tab" aria-controls="metadata" aria-selected="false" onclick="disableButton2()">Metadata</button>
       </li>
       <li class="nav-item">
-        <button class="nav-link" id="nodes-tab" data-bs-toggle="tab" data-bs-target="#nodes" type="button" role="tab" aria-controls="nodes" aria-selected="false">Nodes</button>
+        <button class="nav-link" id="nodes-tab" data-bs-toggle="tab" data-bs-target="#nodes" type="button" role="tab" aria-controls="nodes" aria-selected="false" onclick="disableButton2()" >Nodes</button>
       </li>
+   
+            <button type="button" class="nav-link" data-bs-toggle="dropdown" aria-selected="false" id="results-buttons"  >Results</button>
+            <li class="dropdown-menu">
+              <button class="dropdown-item" data-bs-toggle="tab" id="results-tab" data-bs-target="#results" onclick="disableButton2()">Results All </button>
```

Comment:
instead of a clickable list, use a combobox to select the scenario. The user should select the scenario once and get visual confirmation all the time after that. After it is selected, the scenario automatically applied to the feadback and results pages